src/widgets/setupdialog.cpp

```diff
@@ -392,6 +392,7 @@ void SetupDialog::setTarsnapVersion(QString versionString)
     {
         _ui.clientVersionLabel->setText(tr("Tarsnap CLI version ") +
                                          _tarsnapVersion + tr(" detected.  ✔"));
+        _ui.advancedCLIWidget->hide();
```

Comment:
Doing this has several problems:
1. If Tarsnap CLI is found in default path, when advancing to the Advanced page the widget briefly flickers on and off as the state is changed;
2. If Tarsnap CLI is not found and you proceed to editing the path lineedit, as soon as a valid path is reached the text is changed and the widget is bluntly hidden, leaving you in a state of "What exactly just happened?", even so for a new user;